dvc/stage.py

```diff
@@ -1006,10 +1006,12 @@ def _status(entries):
         return ret
 
     @rwlocked(read=["deps", "outs"])
-    def status(self):
+    def status(self, for_status_command=False):
```

Comment:
Hm, the naming seems a bit cumbersome to me. How about `check_updates`?

Reply:
Works for me!